.github/workflows/release.yaml

```diff
@@ -69,12 +69,12 @@ jobs:
             ARCHIVE="gleam-$VERSION-${{ matrix.build }}.zip"
             cp "target/${{ matrix.target }}/release/gleam.exe" "gleam.exe"
             7z a "$ARCHIVE" "gleam.exe"
-            echo "::set-env name=ASSET::$ARCHIVE"
+            echo "name=ASSET::$ARCHIVE" >> $GITHUB_ENV
           else
             ARCHIVE="gleam-$VERSION-${{ matrix.build }}.tar.gz"
             cp "target/${{ matrix.target }}/release/gleam" "gleam"
             tar -czvf "$ARCHIVE" "gleam"
-            echo "::set-env name=ASSET::$ARCHIVE"
+            echo "name=ASSET::$ARCHIVE" >> $GITHUB_ENV
```

Comment:
Is this right? I can't see the `::` in the docs here?

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#environment-files

Reply:
Silly me, I misunderstood how the previous thing worked (never had a chance to use it and it looks like they provide only new documentation now). I'll fix it soon.

Reply:
Updated, this should do it.